src/ring_theory/ideal/basic.lean

```diff
@@ -459,6 +459,19 @@ protected theorem nontrivial {I : ideal α} (hI : I ≠ ⊤) : nontrivial I.quot
 lemma mk_surjective : function.surjective (mk I) :=
 λ y, quotient.induction_on' y (λ x, exists.intro x rfl)
 
+/--
+The preimage of the image of a set under the quotient map is the union of the cosets covered by `s`.
+-/
```

Comment:
This docstring doesn't mean anything, and it sounds less specific than the statement it documents. I guess it tries to say that the preimage of the image of s is the union of the cosets *intersected* by s. But this would be true for every equivalence relation, whereas this lemma is specific to the equivalence relation associated to an additive subgroup.

Reply:
I'm not too sure how this is a more specific statement; am I missing something mathematically? Or would this better phrasing do?

```lean
/-- The preimage of the image of a set under the quotient map is the union of the additive cosets
that have a representative in `s`. -/
```

Reply:
Did you read the statement of the lemma? Maybe you could have a specific look at the right hand side. There is a union, as in your sentence. But it's not a union of cosets. Maybe it's a notation issue. `(λ y, x.1 + y) '' s` mean the image of the set `s` by the map sending `y` to `x.1 + y`. For instance when `x = 0`, which is indeed in `I`, we get `s` itself. How is that a coset for every set `s`?

Reply:
Ohh!! I understand now, not sure why it didn't make sense to me at all before. I will apply Kevin's suggestion, as I think it makes it clear.